protocol/messenger_communities_import_discord.go

```diff
@@ -1184,12 +1187,20 @@ func (m *Messenger) RequestImportDiscordCommunity(request *requests.ImportDiscor
 			}
 
 			if !exists {
+				channelUniqueName := channel.Channel.Name
+				if count, ok := uniqueChatNames[channelUniqueName]; ok {
+					uniqueChatNames[channelUniqueName] = count + 1
+					channelUniqueName = fmt.Sprintf("%s_%d", channelUniqueName, uniqueChatNames[channelUniqueName])
```

Comment:
I would put count+1 in a separate variable so that it doesn't have to be queried from the map again, but that's more a matter of taste.